lib/presets.js

```diff
@@ -428,6 +428,63 @@ module.exports = {
           return callback(error);
         }
       });
+    },
+
+    /**
+     * @summary Update quoted version immediately following a regex
+     * @function
+     * @public
+     *
+     * @param {Object} options - options
+     * @param {String} [options.baseDir] - directory to prefix onto options.file
+     * @param {String} options.file - file to modify
+     * @param {String} options.regex - regex leading up to the quoted version string
+     * @param {String} [options.regexFlags] - any modifier flags to apply to the regex
```

Comment:
Something in those lines should point the user where to look for more information on regex flags: `modifier flags as used in RegExp`? It is also clear that way that this is the same argument, not something different.